### DISCUSSION TITLE
ask for refund

### DISCUSSION THREAD

**Discussion by bkchopa** [aclap-dev/video-downloadhelper#887] [License & Purchase requests]
### What's your issue?

Hello,

I purchased the Chrome version to download YouTube videos, but it doesn't work. Please refund me.

Instead, I bought the Firefox version.

my email : bkchopa@gmail.com

### The 10 first digits (only 10) of your license number or email address

bkchopa@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by bkchopa** [reply]
thanks!
